solr/core/src/test/org/apache/solr/cloud/TestCloudSearcherWarming.java

```diff
@@ -311,14 +307,11 @@ public boolean onStateChanged(Set<String> liveNodes, DocCollection collectionSta
                     log.error(
                         "registered searcher not null, maxdocs = {}",
                         registeredSearcher.get().maxDoc());
+                    registeredSearcher.decref();
```

Comment:
You tried to improve something but it's no longer correct.  Can't decref then get!  decref last (when done).

Reply:
Restored back to what you had...  thanks!